lib/gis/parser_rest_md.c

```diff
@@ -58,6 +58,10 @@ void usage_rest_md(bool rest)
     fprintf(stdout, "description: %s\n", st->module_info.description);
     fprintf(stdout, "keywords: ");
     G__print_keywords(stdout, NULL, FALSE);
+    fprintf(stdout, "\n");
+    fprintf(stdout, "tags: [ ");
+    G__print_keywords(stdout, NULL, FALSE);
+    fprintf(stdout, " ]");
```

Comment:
Eventually, we can merge tags and keywords, but keywords are indeed just a string while tags are a list, so we will eventually need to switch to parsing YAML list for the keywords to make it work. Let's leave that for later than. The duplication does not harm anything in generated code.